### PR TITLE
[ADD] events: event templates doc

### PR DESCRIPTION
**Pull request by ksc-odoo**
PROJECT TASK: https://www.odoo.com/web#id=3901937&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

This BRAND NEW PR will focus on everything the user should know about Event Templates
- new workflow
- new screenshots
- added to toctree

This PR is also adhering to some recent feedback I've received about getting too granular with explanations, so I am attempting to be more concise with the instructions presented throughout.

*Also, this PR should be merged/published AFTER #8929 , because there is a lot of referenced links used in this PR, which will likely result in continued build errors until that create_events doc is published.*

### CODE REVIEW

File: content/applications/marketing/events/event_templates.rst
Comment (Felicious) [inline content/applications/marketing/events/event_templates.rst:151]: I was thinking about the info in the tip admonition – it's quite lengthy but seems really important. Maybe we could take it out of the tip block since it's not exactly extra info, and a "tip" usually implies something extra.

But since the tip covers various ways of creating a Booth category, I was thinking we could give "Booth Categories" its own dedicated subsection, maybe with an h5 tag (*****). What do you think?

File: content/applications/marketing/events/event_templates.rst
Comment (Felicious) [inline content/applications/marketing/events/event_templates.rst:193]: I like how you included how to use the template after showing how to set it up! (:

File: content/applications/marketing/events/event_templates.rst
Comment (Felicious) [inline content/applications/marketing/events/event_templates.rst:199]:
```suggestion
To use an event template, navigate to the :menuselection:`Events app` and click :guilabel:`New` to open a new event form.
```

**Review by Felicious — APPROVED** [review]
Awesome work on this document, @ksc-odoo! 

It provides a comprehensive overview of the Booth tab, which hasn't been covered in the Events documentation before. I really appreciate how you've kept the document concise by linking to relevant resources, ensuring we avoid any duplicate information across our docs. 

I just had some minor rewording changes for you to consider, so after taking a look at those, feel free to move this onto the next stage of review!

File: content/applications/marketing/events/event_templates.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_templates.rst:74]: I completely understand the suggestion, however, since some readers may not be aware of the concept of a 'submenu' I am going to opt for the slightly more "elaborate" explanation here, just to ensure there is no confusion. That said, if I receive a similar comment about this, I'll be sure to implement your suggestion (and give you full credit 😄 )

File: content/applications/marketing/events/event_templates.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_templates.rst:151]: I opted for an admonition here, because once the brief tangent about Booth Categories is complete, I want the reader to know (both visually and procedurally) that they can pick up the next step process -- I feel like admonitions, like this, provide some much need "boundaries" for instances like this. That said, I agree, its on the "lengthier" side of things when it comes to admonition blocks -- but I think it still works. However, if it gets brought up once again in a review, I'll make some necessary adjustments, and again, give you full credit for pointing it out first 👍

File: content/applications/marketing/events/event_templates.rst
Comment (ksc-odoo) [reply content/applications/marketing/events/event_templates.rst:122]: I added a line after the current paragraph with a link, as suggested. Thanks!

**Comment by ksc-odoo** [comment]
Thanks for the super helpful feedback, @Felicious -- just implemented all your changes, and believe this one is now ready for Final Review, @StraubCreative 👍

File: content/applications/marketing/events/event_templates.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_templates.rst:18]: suggestion: don't need "users" to complete the sentence.
```suggestion
reveals the :guilabel:`Event Templates` page. Here, find all the existing event templates
```

File: content/applications/marketing/events/event_templates.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_templates.rst:25]: Same here: don't need "users" or any other pronouns to complete the sentence.
Consider removing doc-wide.

```suggestion
By default, Odoo provides three pre-configured event templates: :guilabel:`Exhibition`,
```

File: content/applications/marketing/events/event_templates.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_templates.rst:27]: can remove the redundancy and tie the two sentences together 🤞 
```suggestion
:guilabel:`Training`, and :guilabel:`Sport`, which all have their own unique customizations applied to them.
```

File: content/applications/marketing/events/event_templates.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_templates.rst:45]: Suggestion around grammar and presentation:
I'm pretty sure if we begin a sentence with the conjunction "either," we need to pair it with another conjunction like "or" since the word _either_ is meant to present two (or more) options.


Alternatively, these options can be put in numbered list (without the either/or condition), with a quick **bold label** in the beginning, for a different type of presentation:
```suggestion
#. **On the dashboard**, by navigating to :menuselection:`Events app --> Configuration --> Event
   Templates` and clicking the :guilabel:`New` button in the upper-left corner. Doing so reveals a
   blank event template form that can be customized in a number of different ways.
#. **On an event form itself.** Start by typing the name of a new event template in the *Template*
   field, and
```

File: content/applications/marketing/events/event_templates.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_templates.rst:86]: Suggestion around presentation:
The "...**must** create a booth product" part could be an inline admonition block (note, important) inside the list, to further visually highlight the importance of making a product first.

File: content/applications/marketing/events/event_templates.rst
Comment (StraubCreative) [inline content/applications/marketing/events/event_templates.rst:115]: I like what you did with this list 👍

**Review by StraubCreative — APPROVED** [review]
Hi @ksc-odoo 
I thought this was well-written and pleasant to read, nice job 👍 
There's only a few comments for you below but they're all high-level and optional, for you to consider. In any case, approving for the next round, so feel free to tag SALI when you're ready, thanks!

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:91]: @ksc-odoo the build error is most likely due to this :)
```suggestion
  (:menuselection:`Events app --> Configuration --> Booth Categories`).

  .. important::
     First, users **must** create a booth product with the required *Event Booth* option set as the
     :guilabel:`Product Type`on the product form.

- :guilabel:`Exhibitors Menu Item`: adds a menu item that takes visitors to a separate page,
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:31]: You can use the `oi-draggable` icon (_which is coming soon in a PR near you_ https://github.com/odoo/documentation/pull/9256), it will appear once the PR is merged :)
```suggestion
drag-and-drop them into any desired order, using the :icon:`oi-draggable` :guilabel:`(draggable)`
icon, located to the left of each event template line on the :guilabel:`Event Templates` page.
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:44]: nitpick Period after formatting
```suggestion
#. **On an event form itself**. Start by typing the name of a new event template in the *Template*
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:81]: Specificity and refer to submenu consistently :)
```suggestion
- :guilabel:`Website Submenu`: enables a submenu on the event's website. When this checkbox is
  ticked, every other checkbox in this series is automatically ticked, as well. Then, choose to
  untick (deselect) any of the checkbox options, as desired.
- :guilabel:`Tracks Menu Item`: adds a submenu item to navigate to a page displaying all planned
  tracks for the event.
- :guilabel:`Track Proposals Menu Item`: adds a submenu item to navigate to a page, in which
  visitors can fill out a form to propose a track (talk, lecture, presentation, etc.) to happen
  during the event.
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:82]:
```suggestion
- :guilabel:`Booth Menu Item`: adds a submenu item that takes visitors to a separate page, where an
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:89]:
```suggestion
     First, users **must** create a booth product with the required *Event Booth* option set as the
     :guilabel:`Product Type` on the product form.
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:91]:
```suggestion
- :guilabel:`Exhibitors Menu Item`: adds a submenu item that takes visitors to a separate page,
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:94]:
```suggestion
- :guilabel:`Community`: adds a submenu item allowing attendees to access pre-configured virtual
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:101]:
```suggestion
Once the desired checkboxes have been ticked, select an appropriate :guilabel:`Timezone` for the
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:119]: Nitpick to simplify the custom anchor
```suggestion
- :ref:`Booths <event_templates/event_template/booths>`
- :ref:`Questions <events/event-questions>`
- :ref:`Notes <events/event-notes>`

.. _event_templates/event_template/booths:
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:137]: I _think_ you mean just Booths menu here, not Booth Categories :)
```suggestion
   :menuselection:`Events app --> Configuration --> Booth`.
```

File: content/applications/marketing/events/event_templates.rst
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates.rst:174]: Icon label formatting :)
```suggestion
   :guilabel:`(camera)` icon in the upper-right corner.
```

File: content/applications/marketing/events/event_templates/create-booths-popup.png
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates/create-booths-popup.png:1]: Please compress this image :)
`event_templates/create-booths-popup.png`

File: content/applications/marketing/events/event_templates/event-templates-page.png
Comment (samueljlieber) [inline content/applications/marketing/events/event_templates/event-templates-page.png:1]: Please compress this image too :)
`event_templates/event-templates-page.png`

**Review by samueljlieber — APPROVED** [review]
Hi @ksc-odoo! Nice job on this brand new Events doc on Event Templates. I think you covered this feature nicely :)

I am approving with some corrections and suggestions, please make sure to compress the images before merge! Thank you for your hard work on this!
.....
@robodoo delegate=ksc-odoo

**Comment by ksc-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
